multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ValidateDeployParametersStep.java

```diff
@@ -131,16 +149,21 @@ private void deleteFiles(ProcessContext context, List<FileEntry> fileEntries) th
         fileSweeper.sweep(fileEntries);
     }
 
-    private void validateArchive(ProcessContext context) {
+    private List<FileEntry> getArchivePartEntries(ProcessContext context) {
```

Comment:
when the mtar is not split, there is only 1 part, so return it and check if the size is one instead of empty.
then this message can be moved where the check is:
            getStepLogger().infoWithoutProgressMessage(Messages.ARCHIVE_WAS_NOT_SPLIT_TOTAL_SIZE_IN_BYTES_0, archiveFileEntry.getSize());

Reply:
+1